src/app/booking/WholeWeekDialog.js

```diff
@@ -0,0 +1,99 @@
+"use client";
+
+import { OssContainer } from "@/components/OosContainer";
+import {
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  Button,
+  Checkbox,
+  FormGroup,
+  FormControlLabel,
+  Typography,
+} from "@mui/material";
+import { SmFlex } from "@/app/activities/[activityId]/edit/components/responsiveFlexes";
+
+export default function WholeWeekDialog({
+  open,
+  onClose,
+  onConfirm,
+  activity,
+  earlyDropOffDays,
+  setEarlyDropOffDays,
+  latePickupDays,
+  setLatePickupDays,
+}) {
+  const totalPrice = activity?.price * 5;
+  const daysOfWeek = ["M", "T", "W", "T", "F"];
```

Comment:
How does libraries keep day internally? Using single letter is a bad practise. For example, how would you differentiate Sunday and Saturday?

Reply:
daysOfWeek is used for display day. Sherry wants to use 1 letter as a label

Reply:
Displaying data and manipulating data are different layers. You can do something like dayOfWeek[0].upper() to display data.